.circleci/config.yml

```diff
@@ -107,47 +83,87 @@ jobs:
           path: /tmp/artifacts/
   build-docker-image:
     docker:
-      - image: circleci/node:8
+      - image: circleci/buildpack-deps:xenial
```

Comment:
This needs npm below when running `npm run bundle`, we've installed npm inside a `buildpack-deps:xenial` container [in the last rebase](https://github.com/mozilla/redash/blob/b42623e2a106a9e3ec1dbc94715b433ee0b3ad51/.circleci/config.yml#L84-L119) but we can also keep using node:8 and just `sudo apt install python-pip` to get pip which would be more in line with what upstream has.